### ISSUE TITLE
Login-App to tally comments and provide info to client for new comment tracking purposes

### ISSUE THREAD

**Issue by writetoalexander** [Danger-Will-Robinson/mariner#69]

**Comment by BigWidowsPeak** [comment]
This is done. See sample in Slack channel. Will close once it is merged into master.

`    "commentCountByVideoID": {
        "ZvOo6P0dS8o": 1,
        "QsjpYurZ9zc": 1,
        "yy3aj-SMn28": 1,
        "1tMWGC2vrLc": 1,
        "hiP4gxbFbtQ": 1,
        "aV3dC1Sjwhw": 2,
        "CyPNGyJy7XM": 2,
        "eIWNr0Piz1o": 1,
        "CuA6HuZWEU0": 2,
        "3vJmFuPMmBU": 1,
        "zHSfh87GnFc": 1,
        "2kZZ8GnbURA": 1,
        "zLw_XO50yAo": 3,
        "pI0j6tYc7j0": 1,
        "JABL3qfzuBM": 1
    } `
